### PR TITLE
Release version 1.14.0

### PR DESCRIPTION
**Pull request by pschorf**

### CODE REVIEW

File: scheduler/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.14.0] - 2018-03-28
+### Added
+- Support for listing custom executor jobs in /list endpoint, from @dposada
+- Kill instances for cancelled jobs on leadership election, from @pschorf
+### Changed
+- Performance improvements, from @scrosby
+- Fix for GPU support, from @dPeS
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: In the past, we've used "Fixed"

File: scheduler/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.14.0] - 2018-03-28
+### Added
+- Support for listing custom executor jobs in /list endpoint, from @dposada
+- Kill instances for cancelled jobs on leadership election, from @pschorf
+### Changed
+- Performance improvements, from @scrosby
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Should we be more specific about the areas improved?

File: scheduler/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.14.0] - 2018-03-28
+### Added
+- Support for listing custom executor jobs in /list endpoint, from @dposada
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: This is acually in `/jobs`, `/list` remained unchanged

File: scheduler/CHANGELOG.md (hunk 2 above)
Comment (pschorf) [reply scheduler/CHANGELOG.md]: Done

File: scheduler/CHANGELOG.md (hunk 3 above)
Comment (pschorf) [reply scheduler/CHANGELOG.md]: Done

**Pull request merged**